.github/workflows/build.yml

```diff
@@ -298,3 +298,73 @@ jobs:
         if-no-files-found: error
         retention-days: 30
 
+  archlinux:
+    strategy:
+      matrix:
+        arch:
+          [
+            "linux/amd64 x86_64",
+            "linux/arm64/v8 aarch64",
+          ]
+    name: "Archlinux ${{ matrix.arch }}"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        version: latest
+    - name: Read info
+      id: tags
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        echo ::set-output name=PLATFORM::${arch%% *}
+        echo ::set-output name=ARCH::${arch##* }
+    - name: set environment variables
+      id: set_vars
+      run: ./scripts/ci-set-vars.sh
+      env:
+        REPOSITORY: ${{ github.event.repository.name }}
+    - name: Build ${{ matrix.arch }} release
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        PLATFORM=${arch%% *}
+        ARCH=${arch##* }
```

Comment:
`ARCH` should also be available via `steps.tags.outputs.ARCH`, analogous for `PLATFORM`.